internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -80,6 +79,23 @@ func resourceDatadogSingleSignOnConfigurations() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOh() {
+		resource.Schema["single_sign_on"].Required = false
+		resource.Schema["single_sign_on"].Optional = true
+		resource.Schema["single_sign_on_enabled"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(singlesignon.SingleSignOnStatesEnable),
+				string(singlesignon.SingleSignOnStatesDisable),
+			}, false),
+			ExactlyOneOf: []string{"single_sign_on", "single_sign_on_enabled"},
+			Deprecated:   "This property is deprecated and will be removed in v5.0 of the AzureRM provider. Please use the `single_sign_on` property instead.",
```

Comment:
We should also keep our deprecation messages consistent

```suggestion
			Deprecated:   "`single_sign_on_enabled ` has been deprecated in favour of the `single_sign_on ` property and will be removed in v5.0 of the AzureRM Provider.",
```

`single_sign_on_enabled ` has been deprecated in favour of the `single_sign_on ` property and will be removed in v5.0 of the AzureRM Provider.